<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>941</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_1">
       <attribute name="title">
        <string>조회개요</string>
       </attribute>
       <widget class="QComboBox" name="indus_cb">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>100</y>
          <width>211</width>
          <height>22</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>실내건축공사업</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>건축공사업</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>기계설비,가스공사업</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>전문소방시설공사업</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>일반소방시설공사업기계</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>일반소방시설공사업전기</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>상ㆍ하수도설비공사업</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>금속창호ㆍ지붕건축물조립공사업</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>시설물유지관리업</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>철근ㆍ콘크리트공사업</string>
         </property>
        </item>
       </widget>
       <widget class="QLineEdit" name="presmptPrceEnd">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>200</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>194,000,000</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="days">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>240</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>30</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="presmptPrceBgn">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>160</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>129,000,000</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>160</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>추정가(하한)</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>200</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>추정가(상한)</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>240</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>일간</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>240</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>검색기간</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>110</y>
          <width>56</width>
          <height>12</height>
         </rect>
        </property>
        <property name="text">
         <string>면허</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>50</y>
          <width>56</width>
          <height>12</height>
         </rect>
        </property>
        <property name="text">
         <string>지역</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="region">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>40</y>
          <width>141</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>인천광역시</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>500</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>조회</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>330</y>
          <width>141</width>
          <height>141</height>
         </rect>
        </property>
        <property name="title">
         <string>기타</string>
        </property>
        <widget class="QCheckBox" name="valuA_chk">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>40</y>
           <width>82</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>A 값</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>100</y>
           <width>111</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>순공사원가적용</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>검색자료</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>6</number>
            </property>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btn_analyze">
              <property name="styleSheet">
               <string notr="true">석</string>
              </property>
              <property name="text">
               <string>분석</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableView" name="table_view">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>확률그래프</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>941</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
